deploy/development/secret.tpl

```diff
@@ -34,3 +34,11 @@ type: Opaque
 data:
   AWS_CLOUDFRONT_PUBLIC_KEY: "${AWS_CLOUDFRONT_PUBLIC_KEY_BASE64}"
   # AWS_CLOUDFRONT_PUBLIC_KEY_EXPIRING: "${AWS_CLOUDFRONT_PUBLIC_KEY_EXPIRING_BASE64}"
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: basic-auth-secret
+type: Opaque
+data:
+  auth: "${BASIC_AUTH_BASE64}"
```

Comment:
This will be the base64 encoded string with the hashed password.

On justice dev I didn't need the un-hashed password in the application. I think the user and pass should be added as 2 new lines around line number 19 in this file.

Then they'll be available as env vars to php :)